enterprise_access/apps/api/exceptions.py

```diff
@@ -0,0 +1,17 @@
+"""
+Custom Exception classes for Enterprise Access API v1.
+"""
+
+class SubsidyRequestError(Exception):
+    """
+    A general exception dealing with subsidy requests.
+    """
+    def __init__(self, message, http_status_code=None):
+        super().__init__(message)
+        self.message = message
+        self.http_status_code = http_status_code
+
+class SubsidyRequestCreationError(SubsidyRequestError):
+    """
+    An exception indicating that a subsidy request cannot be created.
+    """
```

Comment:
Does this pass the linter? Should there be something different about these two classes? If they are the same, do we need both?

Reply:
It does. We don't need both, if we want to distinguish between different types of subsidy request errors this would be helpful.